packages/webapp/src/components/Finances/Transaction/ExpandedContent/CropSaleTable.jsx

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Table from '../../../Table/v2';
+import history from '../../../../history';
+import styles from './styles.module.scss';
+
+const getColumns = (t, mobileView, totalAmount, quantityTotal, currencySymbol) => [
+  {
+    id: 'title',
+    label: t('FINANCES.TRANSACTION.CROPS'),
+    format: (d) =>
+      mobileView ? (
+        <div className={styles.mobileCrops}>
+          <div className={styles.mobileCropsTitle}>{d.title}</div>
+          <div className={styles.mobileCropsQuantity}>
+            {d.quantity} {d.quantityUnit}
+          </div>
+        </div>
+      ) : (
+        d.title
+      ),
+    columnProps: {
+      style: { padding: `0 ${mobileView ? 8 : 12}px` },
+    },
+    Footer: mobileView ? null : t('FINANCES.TRANSACTION.DAILY_TOTAL'),
+  },
+  {
+    id: mobileView ? null : 'quantity',
+    label: mobileView ? null : t('common:QUANTITY'),
+    align: 'right',
+    format: (d) => `${d.quantity} ${d.quantityUnit}`,
+    columnProps: {
+      style: { width: '100px' },
+    },
+    Footer: mobileView ? null : <div className={styles.bold}>{quantityTotal}</div>,
+  },
+  {
+    id: 'amount',
+    label: t('FINANCES.REVENUE'),
+    align: 'right',
+    format: (d) => `${currencySymbol} ${Math.abs(d.amount).toFixed(2)}`,
+    columnProps: {
+      style: { width: '250px', paddingRight: `${mobileView ? 9 : 75}px` },
```

Comment:
Only noticed because I was to struggling to find the right column widths for my own table, but this seems like maybe too much width for mobile as it truncates the first column down to almost nothing. (I see passing the FooterCell separately fixes truncation there though... neat!)

<img width="362" alt="Screenshot 2023-11-02 at 10 46 27 AM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/040f00d2-1341-4fb5-8416-7c2b1f0e28e8">